gcn/notices/core/Localization.schema.json

```diff
@@ -34,6 +30,46 @@
       "description": "Contains true when the systematic error is included and false when the systematic error is not included",
       "default": false
     },
+    "position_angle": {
+      "type": ["number", "null"],
+      "description": "Position angle of semi-major axis [deg]"
+    },
+    "galactic_l": {
+      "type": ["number", "null"],
+      "description": "Galactic longitude [deg]"
+    },
+    "galactic_b": {
+      "type": ["number", "null"],
+      "description": "Galactic latitude [deg]"
+    },
+    "galactic_semimajor_angle": {
+      "type": ["number", "null"],
+      "description": "Position angle of semi-major axis in galactic coordinates [deg]"
+    },
```

Comment:
I would rather not include galactic coordinates as we can easily calculate them from other frames (such as ICRS). I understand the convenience of calculating them for users, but we need not include them in the alerts themselves in order to do that.

Reply:
You mentioned that at the meeting, then Judy said, it is a useful piece of info to have at hand. I put it in, because it might help with your team's decision.

Reply:
I don't have a strong opinion either way on this.

Reply:
I do. It's easily derived from ICRS coordinates. Let's omit the Galactic coordinates in order to decrease the surface area of the schema that missions have to implement. We can easily provide Galactic coordinates and many other coordinate frames through APIs and libraries.